x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/host/host_metrics_table.tsx

```diff
@@ -67,38 +59,50 @@ export const HostMetricsTable = (props: HostMetricsTableProps) => {
     [setSortState, setCurrentPageIndex]
   );
 
-  if (isLoading) {
+  if (data.state === 'error') {
```

Comment:
Since this is the same across the 3 tables, I wonder if it's helpful to have some common function for this? 

So far most of what the tables do is duplicated, so I still struggle with deciding when it to share code and when not to. But many of our PRs here are "make the same change in 3 components".